_adr/80/index.adoc

```diff
@@ -1,22 +1,90 @@
 ---
 num: 80
-title: "CEE diagnostic gathering mechanism"
+title: "Diagnostic Gathering"
 status: "Draft"
 authors:
   - "Keith Wall"
 ---
 
 ## Context and Problem Statement
 
-In order to provide support to customers of a Managed Service, CEE require a mechanism that allows them to capture the
-state of the instance of the service for diagnostic purposes.  CEE will use this information in conjunction with the
-problem statement provided by the customer in order to efficiently investigate the problem and determine a solution.
+In order to provide support to the customers of a Managed Service, a company offering the Managed Service will typically have a Support
+Organisation (SO) dedicated to this function.  In order to perform this function effectively the SO requires a mechanism that allows
+them to capture the state of the instance of the service for diagnostic purposes.
 
-The data gathering mechanism must consider the issue of Personally Identifiable Information (PII) or other sensitive
-information and ensure that it is handled appropriately.
+The SO will use the gathered information in conjunction with the problem statement provided by the customer in order to efficiently
+investigate the problem and determine a solution.
+
+The data gathering mechanism must consider the issue of Personally Identifiable Information (PII) or other sensitive information and ensure
+that it is handled appropriately.
 
 Authorization controls must be in place to ensure that the diagnostic gathering mechanism can only be used by suitably
 permissioned groups.
 
+## Goals
+
+* Identify the preferred solution for the diagnostic gathering mechanism.
+* Provide guidelines for content that should be included.
+
+## Non-goals
+
+None
+
+## Stakeholders
+* Eng
+* BU
+* SRE
+* SO
+
+## Current Architecture
+
+None
+
+## Proposed Architecture
+
+https://datatracker.ietf.org/doc/html/rfc2119[RFC-2119] keywords are used in this proposal.
+
+It is RECOMMENDED that managed services use https://source.redhat.com/groups/public/openshiftplatformsre/wiki/backplane_scripts[Back Plane]
```

Comment:
Non-public link

Reply:
Fixed.

Reply:
It's still a non-public link, did you push your changes?

Reply:
Not fixed, unresolving.